src/routes/wishes.routes.ts

```diff
@@ -5,16 +5,25 @@ import {
   clearWishlist,
   clearOneWishlistItem,
 } from "../controllers";
-import { isAuth } from "../middleware/auth";
+import { AuthAndRoleChecker } from "../middleware/auth/authanticated.middleware";
+import { B_ACCESSKEY } from "../utils/roleConstants";
```

Comment:
I want to point out that you are currently using hardcoded roles `B_ACCESSKEY, A_ACCESSKEY, and S_ACCESSKEY.` However, I don't think this will work properly if the admin assigns a role ID to a user that is different from these hardcoded ones.

Reply:
Thank you @emilenfc ,  
but If you have just checked the suggested changes from @mucyoalexaxel, he has just suggested that instead of using environmental variable.